app/controllers/users.js

```diff
@@ -155,20 +160,77 @@ exports.signUp = (req, res) => {
   }
 };
 
+exports.sendCredentials = (req, res) => {
+  if (req.params.token) {
+    const { token } = req.params;
+    jwt.verify(token, secret, (err, decoded) => {
+      if (err) {
+        return res.status(401).json({ success: false, message: 'Activation link has expired' });
+      }
+      User.findOne({
+        /* eslint-disable */
```

Comment:
You should reenable eslint with `/* eslint-enable */` after the line(s) you wish to disable @ddouglasz